design-documents/graph-ql/coverage/Cart.graphqls

```diff
@@ -11,22 +11,138 @@ type CartQueryOutput {
+    links_v2: [DownloadableProductLink] @doc(description: "An array containing information about the selected links")
+    samples: [DownloadableProductSamples] @doc(description: "DownloadableProductSamples defines characteristics of a downloadable product")
+}
 
-    gift_cards_amount_used: Money
-    applied_gift_cards: [CartGiftCard]
+type BundleCartItem implements CartItemInterface {
+    bundle_options: [SelectedBundleOption!]
+}
 
-    is_multishipping: Boolean!
-    is_virtual: Boolean!
+type GiftCardCartItem implements CartItemInterface {
+    sender_name: String!
+    recepient_name: String!
+    amount: SelectedGiftCardAmount
+    message: String
+}
+
+type SelectedConfigurableOption {
+    # Hash which includes option ID, option type and slected values. Can be used to move configurable product to wishlist or gift registry
+    id_v2: ID!
+    id: Int!
+    option_label: String!
+    value_id: Int!
+    value_label: String!
+}
+
+type DownloadableProductLink @doc(description: "Defines characteristics of a downloadable product") {
+    # Hash based on option type and slected link. Can be used to move downloadable product to wishlist or gift registry
+    id: ID!
+    title: String @doc(description: "The display name of the link")
+    sort_order: Int @doc(description: "A number indicating the sort order")
+    price: Float @doc(description: "The price of the downloadable product")
+    sample_url: String @doc(description: "URL to the downloadable sample")
+}
+
+type DownloadableProductSamples @doc(description: "DownloadableProductSamples defines characteristics of a downloadable product") {
+    title: String @doc(description: "The display name of the sample")
+    sort_order: Int @doc(description: "A number indicating the sort order")
+    sample_url: String @doc(description: "URL to the downloadable sample")
+}
+
+type SelectedBundleOption {
+    id: Int!
+    label: String!
+    type: String!
+    values: [SelectedBundleOptionValue!]!
+}
+
+type SelectedBundleOptionValue {
+    id: Int!
+    label: String!
+    price: Float!
+    quantity: Float!
+    child_sku: String!
+}
+
+type SelectedCustomizableOption {
+    # Hash which includes option ID, option type and slected values. Can be used to move a product to wishlist or gift registry
+    id_v2: ID!
+    id: Int!
+    label: String!
+    is_required: Boolean!
+    values: [SelectedCustomizableOptionValue!]!
+    sort_order: Int!
+}
+
+type SelectedCustomizableOptionValue {
+    id: Int!
+    label: String!
+    value: String!
+    price: CartItemSelectedOptionValuePrice!
+}
+
+type CartItemSelectedOptionValuePrice {
+    value: Float!
+    units: String!
+    type: PriceTypeEnum!
+}
+
+type SelectedGiftCardAmount {
+    # Hash from the type of the option and value
+    id: ID!
+    value: Money!
+}
+
+enum PriceTypeEnum @doc(description: "This enumeration the price type.") {
```

Comment:
Description on this one isn't super clear, but probably doesn't need one (just repeats what the name of the type already says)

Reply:
Again, taken from existing schema as is.. Will correct here